### ISSUE TITLE
Don't mess up comment ticket styles

### ISSUE THREAD

**Issue by toebu** [trac-hacks/trac-mentions#9]
The plugin seems to mess up the styles of the comment ticket view: the wysywig / textarea radio button is in the comment window and cannot be clicked.
![wysywigconfused](https://cloud.githubusercontent.com/assets/1837568/6777597/59ff35dc-d14b-11e4-8e5c-016811eac79f.png)

More annoyingly, it sometimes adds huge amounts of white-space at the bottom of the page. The white-space is gone when you reload the page, but is there again when you open another ticket (although not always, it seems quite random).
![tracbottomwhitespace](https://cloud.githubusercontent.com/assets/1837568/6777600/63f6cf64-d14b-11e4-8275-d02450646b58.png)

I quickly investigated where the white-space comes from: if you remove the wysywig-view <iframe> within the comment area, the white-space is gone. Also, it seems like it continuously updates the <body> of the iframe, even if you do nothing on the page.

**Comment by rjollos** [comment]
You've opened this ticket against trac-mentions plugin. Have you confirmed that the issue is not present when trac-mentions is disabled? The issue could be associated with another plugin.

**Comment by toebu** [comment]
Yep, the issue appeared at the time we enabled trac-mentions and trac-hashtags. Disabling trac-mentions removes the issue, enabling it again brings it back. So pretty sure it's the right plugin.
